README.md

```diff
@@ -265,6 +265,19 @@ by the chosen storage driver's provider. A list of drivers can be found [here](h
 [Horizontal](https://docs.openshift.com/container-platform/latest/nodes/pods/nodes-pods-autoscaling.html) and
 [Vertical](https://docs.openshift.com/container-platform/latest/nodes/pods/nodes-pods-vertical-autoscaler.html) Pod
 autoscaling support are not available for Windows workloads.
+
+### Node reboot in AWS
+
+In AWS, Windows nodes are not able to become ready after [performing a graceful reboot](docs/TROUBLESHOOTING.md#rebooting-a-windows-node) 
+due to an inconsistency with the EC2 instance metadata routes and the HNS networks.
+
+The cluster administrator must SSH into the instance and add the route after rebooting with the following command:
+```cmd
+  route add 169.254.169.254 mask 255.255.255.255 <gateway_ip>
+```
+where `169.254.169.254` and `255.255.255.255` are the address and network mask of the EC2 instance metadata endpoint,
+respectively. The `<gateway_ip>` must be replaced by the corresponding IP address of the gateway in Windows instance. 
```

Comment:
any guidance on how to get `gateway_ip` value? Is it the default gateway e.g. `ipconfig | findstr /C:"Default Gateway"`?

Reply:
Sure, I'm happy to provide general guidance on fetching the `gateway_ip` , but I'm a bit skeptical about a specific script that may fail due to custom configurations.

Reply:
I would add some guidance (or a script that would work with "default" configurations)

Reply:
Addressed in https://github.com/openshift/windows-machine-config-operator/pull/2291/commits/2f419771bbe99f8c5b83de525d7151f6c199cb36#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R282